### PR TITLE
feat: add github corner link to the site

### LINKED ISSUE #14: Add GitHub icon to the corner
So that one can easily click through to the GitHub repository.

### PR DESCRIPTION
**Pull request by Polostor**
# Short description

New github link added to the corner of the website as requested in #14.

Closes #14.

# Changes

feat: add github link to the site #14

# Static visuals

image

# Used

Used implementation by great [Tim Holman](http://tholman.com/) from https://tholman.com/github-corners/

### CODE REVIEW

File: sass/styles.scss [hunk 1]
```diff
@@ -1,3 +1,4 @@
+@import "github";
```
Comment (Indy2222) [inline sass/styles.scss:1]: Nitpic: an empty line below this one would be nice.

File: sass/github.scss [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+
+@media (max-width: 500px) {
+    .github-corner:hover .octo-arm {
+        animation: none
+    }
+
+    .github-corner .octo-arm {
+        animation: octocat-wave 560ms ease-in-out
+    }
+}
```
Comment (Indy2222) [inline sass/github.scss]: Please make sure that all files end with a new-line (according to the definition of line in POSIX).

**Review by Indy2222 — CHANGES_REQUESTED** [review]
Two small nitpicks, otherwise LGTM.

File: sass/styles.scss (hunk 1 above)
Comment (Polostor) [reply sass/styles.scss:1]: added new line

File: sass/github.scss (hunk 2 above)
Comment (Polostor) [reply sass/github.scss]: added new line

**Review by Indy2222 — APPROVED** [review]

**Pull request merged**